kernel/config.txt

```diff
@@ -21,12 +21,144 @@ necessary. Information about pre-built hardware as a whole will not be covered.
+
+The first method is:
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ make menuconfig                                                          |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+which compiles and launches a terminal program in nostalgic colors, and allows
+you to browse the existing configuration, compartmentalised in trees and
+subtrees, using your keyboard and search for and change the configurations you
+want.
+
+Another terminal tool is:
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ make nconfig                                                             |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+which is pretty similar, but different controls and in largely forgiving color
+schemes.
+
+Some GUI configuration tools are:
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ make xconfig                                                             |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+and
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ make gconfig                                                             |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+which require the QT and GTK framworks respectively, and of course, a graphical
+X server running.
+
+Another option you may find very useful to easily trim down general (distro,
+etc.) configuration files is
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ make localmodconfig                                                      |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+It modifes the current .config to only compile whatever drivers are loaded in
+the host kernel's current state. Running this after connecting all the hardware
+you will be using is a pretty quick way to come up with a pretty lean
+configuration.
```

Comment:
This can also be omitted. KISS doesn't use an initramfs and this will likely create a broken config.